Libplanet/Net/Swarm.cs

```diff
@@ -83,6 +82,8 @@ public partial class Swarm<T> : IDisposable
             _privateKey = privateKey ?? throw new ArgumentNullException(nameof(privateKey));
             LastSeenTimestamps =
                 new ConcurrentDictionary<Peer, DateTimeOffset>();
+            _txSyncTaskQueue = new ConcurrentDictionary<BoundPeer, HashSet<TxId>>();
+            _txIgnoreQueue = new FixedSizedQueue<TxId>(100);
```

Comment:
Is there any reason for separated `_txIgnoreQueue`? IMO `stagePolicy.Ignore()` does similar things.